notebooks/optuna-example.ipynb

```diff
@@ -0,0 +1,554 @@
nFjt8QMrUaQmAbSJlZkG7dgkythsuDY3gj5D005n1be6YYh0o8EFCah9a6fOS7e+H7kH647hOIyE6tmm0ca50PAV7uCkR08EG+rhI+Hgr0bOOJRSPamV0Da08/C8g2OAecn9edfym8S4sqHXF3d4dOp0N5eXmDH+5iLjc3t1of0EJNI1VUQCosROH+Q9DtS4Sr3AW3lL4odPWETu6KrlIJZPXcjNCmV9cWtZzf3RShoUyoGqGh71tOcQWu+wTiuo/pXFTfo57XWp8P50HjsAaWiBqrRSXagiAYHwxjKbzDvPlIkgQxPQOGy5cg3kwFAOScPocsD19keaqN+3nqy6AP9LNVmERGrOclqsK1x4mqtKhEmxyDVF4OQ0oKDJcvQyosguDuDkXP+yDv0gXBvfsjZ8MOeOrLTNbEbin11mTfLPGgKyJHU9vazhcySri2M7VITLTJbog5OTBcvgzD79cAgwEy/9ZQ9O4NWbt2xnrrNr26Ai9O/c9THgNb3lMeyX4F+rhh/cOdsOV4Bgr0gI8reCWPWpwtxzNMkmwASCvUY8vxDJbgUIvDRJtsSqqshHj9OgyXLkPMyQFcFJB3CoU8LAwytbrWPi293prsW3U9L8vIqKXi2uNE/8FEm2xCLCyE4fIViCnJkMr1EFr5VK0YEtoRgqvrvQ9A5ERYz0rOxJr3KvB7h+wdE22yGkkUIaamVd3cmJYOyATI27aDvGsYBH9/i60EQ2TPWM9KzsZa9yrwe4ccARNtsjiprAyG5GQYLl+BVFICQekBRZ/ekHfuDEGptHV4RDbFelZyNnfeq5BTUgGtp2WuNPN7hxwBE22yCEmSIN26VXVz4x9/AAYRssA2UPTvB1lwsPHmRqKWjvWs5IyssfY4v3fIEVgt0U5KSsL27dshiiKio6MRGxtr8npxcTE2b96MrKwsuLi44LnnnkPbtm3N6kv2Q6qogHjtGiovXYaUlwe4ukDepQvkXbtC5uNj6/CIGsQa9Z+sZyVqHK5bT9Xs+WebVRJtURQRFxeHRYsWQaPRYP78+QgPD0dwcLBxn127dqF9+/aYO3cu0tLSEBcXh8WLF5vVl2xPzM+vunp99Sqgr4CgVsNl0F8g69ABggt/6JHjsVb9J+tZiRqH69YTYP8/22TWGCQlJQUBAQHw9/eHQqHAoEGDcOrUKZN9UlNT0bNnTwBAUFAQsrOzkZ+fb1Zfsg3JYIDh+nXo9+2D/pvdMFy5AnlwCFwfHAPXcQ9B3qULk2xyWPXVfzan6nrWkWG+6BuswsgwX4v8D8Ja50NkLdb63iH7Zu8/26xyRTsvLw8ajca4rdFokJycbLJPu3btcOLECXTt2hUpKSnIzs5GXl6eWX2rxcfHIz4+HgCwcuVKaLVaC5xN/RQKhU3GtSaxuBjlv/2G8osXIZaUws3LC27Do+DWtStkvLkRQMuYB86uoPx67e16mPXZNmQOaLXAh6FBDQmvwZp6PtQ4/FlgWdb43mkqzgHLsvefbVZJtCVJqtF291JusbGx2LFjB+bOnYu2bduiQ4cOkMlkZvWtFhMTg5iYGOO2LR4W4SwPqcg4c+k/T1/U+KHXIyPh798KlZcvQ7xxA5AAWVAg5PffD1lQECplMpSUlgKlpbYO3S44yzxoyeq6KObjat7PFnubA009n4aw53pJa7O3ecDPxvrsbQ44G2v+bKtLYGBgna9ZJdHWaDTIzc01bufm5sLX19dkH6VSieeffx5AVWL+wgsvoHXr1tDr9ffsS80r48wlJG3YgSzJDeUKD7S59gcyV6yGywM94NU+GPLu3aEIC4Pg5WXrUIksxtnqP1kLTvxsyBnZ+89qq9Roh4aGIiMjA7du3UJlZSWOHTuG8PBwk31KSkpQWVkJADh48CC6desGpVJpVl9qXme+2ocsyQ2elTp0vf0HfMuLUCC44mJWKVwffRQu4eFMssnpOVv9J2vBiZ8NOSN7/1ltlSvacrkc06ZNw4oVKyCKIqKiohASEoL9+/cDAEaOHIm0tDR8+OGHkMlkCA4OxrPPPltvX7Ic19xslCs80LEwHcUuSmR6qqGTu6JreQnXv6YWxRprAVsT1zZu2fjZkLOy55/VVltHu2/fvujbt69J28iRI43/7tKlCz744AOz+5Ll6DV+8Lt+EwKALKUvyhRu8NSXQR/oZ+vQqJFYl0nWwrWN7Rc/GyLrs0rpCDmWXhNHIaiyCDJJhE7hCk99GfyFcvSaOMrWoVEjVNdl7r98G7+mFWP/5duYvSsF6QXltg6NnNAzA9sgyNvVpM2e6iVbMn42RNbHR7BTDQE9u8ClXw/8npyGMKEU+kA/9Jo4Cm16dbV1aNQI9dVl2uuf2shxVddLbjmegZySCmg9+RcUe8HPhsj6mGhTDVJuLlRqHzww9yHIO3a0dTjURKzLJGuz53rJlo6fDZF1sXSEahBvpgKCAFmQfT8EgMzDukwiIiLbYKJNNRjSUiFr7QfBjX9OdAasyyQiIrINlo6QCam0FFJuHuR9+9g6FGomrMskIiKyDSbaZEJMTQUAyIKDbRwJNSfWZRIREVkfS0fIhCEtDYKnEgIfc09ERETUJEy0yUgyGCCmp0MWHAxBEGwdDhEREZFDY6JNRlJWFlBRCVkQy0aIiIiImoqJNhkZUlMBuRyyNgG2DoWIiIjI4THRJiMxNQ2yAH8ILlxfmYiIiKipmGgTAEAsKIBUWAhZcIitQyEiIiJyCky0CQAgpqUBAOTBfBokERERUXNgok0AqspGBB8fCF5etg6FiIiIyCkw0SZIFRUQszL5kBoiIiKiZsREmyBmZAAGkWUjRERERM2IiTZBvJkKuLpAaN3a1qEQEREROQ0m2i2cJEkQ01IhCwyEIJfbOhwiIiIip8FEu4WT8vIglZZBzvpsIiIiombFRLuFE1OrlvWTBbE+m4iIiKg5MdFu4cS0VMi0GggeHrYOhYiIiMipMNFuwSSdDmJ2Dpf1IyIiIrIAJtotmJiWBkgSy0aIiI
```

Comment:
Unrelated note: In general, I have found parallel coordinate plots more useful when they are interactive. (Something like https://facebookresearch.github.io/hiplot/)

Reply:
True, maybe worth suggesting to the optuna maintainers.

Reply:
Optuna now have the interactive [parallel coordinate plot].(https://optuna.readthedocs.io/en/stable/reference/visualization/generated/optuna.visualization.plot_parallel_coordinate.html)

Reply:
Nice, wish come true, thanks for letting us know.